backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  applications               Applications? @relation(fields: [applicationId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  householdMemberAddress     Address?      @relation("household_member_address", fields: [addressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  householdMemberWorkAddress Address?      @relation("household_member_work_address", fields: [workAddressId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@index([applicationId])
+  @@map("household_member")
+}
+
+// Note: [name] formerly max length 256
+model Jurisdictions {
+  id                          String                   @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt                   DateTime                 @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt                   DateTime                 @updatedAt @map("updated_at") @db.Timestamp(6)
+  name                        String                   @unique()
+  notificationsSignUpUrl      String?                  @map("notifications_sign_up_url")
+  languages                   LanguagesEnum[]          @default([en])
+  partnerTerms                String?                  @map("partner_terms")
+  publicUrl                   String                   @default("") @map("public_url")
+  emailFromAddress            String?                  @map("email_from_address")
+  rentalAssistanceDefault     String                   @map("rental_assistance_default")
+  enablePartnerSettings       Boolean                  @default(false) @map("enable_partner_settings")
+  enableAccessibilityFeatures Boolean                  @default(false) @map("enable_accessibility_features")
+  enableUtilitiesIncluded     Boolean                  @default(false) @map("enable_utilities_included")
+  amiChart                    AmiChart[]
+  multiselectQuestions        MultiselectQuestions[]
+  listings                    listings[]
+  reservedCommunityTypes      ReservedCommunityTypes[]
+  translations                Translations[]
+  user_accounts               UserAccounts[]
+
+  @@map("jurisdictions")
+}
+
+model ListingEvents {
+  id        String            @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt DateTime          @default(now()) @map("created_at") @db.Timestamp(6)
+  updatedAt DateTime          @updatedAt @map("updated_at") @db.Timestamp(6)
+  type      MarketingTypeEnum
+  startDate DateTime?         @map("start_date") @db.Timestamptz(6)
+  startTime DateTime?         @map("start_time") @db.Timestamptz(6)
+  endTime   DateTime?         @map("end_time") @db.Timestamptz(6)
+  url       String?
+  note      String?
+  label     String?
+  listingId String?           @map("listing_id") @db.Uuid
+  fileId    String?           @map("file_id") @db.Uuid
+  assets    Assets?           @relation(fields: [fileId], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

Comment:
I think I'm just not understanding these, an asset can't exist on more than one listing, so not sure why we would have listings here, or what assets is related to

Reply:
sure, so:
```
  fileId    String?               @map("file_id") @db.Uuid
  assets    Assets?               @relation(fields: [fileId], references: [id], onDelete: NoAction, onUpdate: NoAction)
```

that fileId is really an asset's id 
then the assets relationship uses that id to make the connection between a listing event and an asset 

because of the structure here it is a 1 to 1 relationship as opposed to a 1 to many or many to many relationship

hope that helps!